pevm/Cargo.toml

```diff
@@ -0,0 +1,80 @@
+[package]
+name = "pevm"
+version = "0.1.0"
+edition = "2021"
+
+[features]
+defaults = []
+rpc = [
+  "tokio",
```

Comment:
These should be `dep:*`? Nit but let's also sort the deps alphabetically and add a newline to separate each feature set.